### PR TITLE
Js additions to fix the button-problem

### PR DESCRIPTION
**Pull request by atlantees**
- Added local storage of sessionActive, date-object for the time of student-feedback and functionality to make sure buttons stay disabled through page refresh and change of session-status.

- Included socket.io in js-directory and updated references.

### CODE REVIEW

**Comment by atlantees** [comment]
The socket.io js appear to have lost any reasonable formatting :weary:

File: iris/static/js/student.js [hunk 1]
```diff
@@ -3,6 +3,13 @@ const buttonFast = document.getElementById("fast");
 const buttonEasy = document.getElementById("easy");
 const buttonHard = document.getElementById("hard");
 const timeOut = 15000;
+var paceClicked = false;
```
Comment (eirism) [inline iris/static/js/student.js]: Should we be consistent in our use of const/let and var?

File: iris/static/js/student.js [hunk 2]
```diff
@@ -21,23 +27,69 @@ function disableAllButtons() {
...
 
+if(!sessionActive) {
+    disableAllButtons();
+}
+
 socket.on('student_recv', function (msg) {
     console.log(msg['active']);
     if(msg.hasOwnProperty('active')) {
-        if(msg['active']) {
+        sessionActive = msg['active'];
```
Comment (Mattis52) [inline iris/static/js/student.js:37]: Anyone know why the sessionActive variable doesn't update correctly?

File: iris/static/js/student.js [hunk 3]
```diff
@@ -3,7 +3,13 @@ const buttonFast = document.getElementById("fast");
...
 const buttonHard = document.getElementById("hard");
 const timeOut = 15000;
-
+let paceClicked = false;
+let difficultyClicked = false;
+let paceTimeOut;
+let difficultyTimeOut;
+let currentTime;
+let timePace;
+let timeDifficulty;
```
Comment (Mattis52) [inline iris/static/js/student.js:12]: It's not necessary to declare all these up here when they (at least some of them) are also being declared further down in the code, cf. standardJS.

**Review by eirism — APPROVED** [review]
Works as intended locally.

**Pull request merged**